engine/time/src/main/java/io/deephaven/time/DateTimeUtils.java

```diff
@@ -981,6 +982,21 @@ public static long epochNanos(@Nullable final ZonedDateTime dateTime) {
         return safeComputeNanos(dateTime.toEpochSecond(), dateTime.getNano());
     }
 
+    /**
```

Comment:
In case my comments seem strong, this library is very exposed to users, so it needs to be ultra currated.  Functions should only get added when there is a compelling reason.

1. I am not a fan of having hard-coded methods for any specific timezone.  
2. The methods should accept a time zone as an input.
3. If methods are added for `LocalDateTime`, `LocalDateTime` signatures should be added to all relevant methods.